app/controllers/catalog_controller.rb

```diff
@@ -18,11 +18,13 @@ def self.modified_field
   include BlacklightIiifSearch::Controller
 
   configure_blacklight do |config|
+    # IiifPrint index fields
+    config.add_index_field 'all_text_tsimv', highlight: true, helper_method: :render_ocr_snippets
```

Comment:
What does this do?  Will this add context snippets to catalog search results views, or is this related somehow to UV search term highlighting?